cms/toolbar/utils.py

```diff
@@ -117,14 +120,41 @@ def get_toolbar_from_request(request):
     return getattr(request, 'toolbar', EmptyToolbar(request))
 
 
+def add_live_url_querystring_param(obj, url, language=None):
+    """
+    Get CMS_ENDPOINT_LIVE_URL_QUERYSTRING_PARAM as the parameter, and the live url, if the object is
+    PageContent
+    """
+    url_param = getattr(settings, 'CMS_ENDPOINT_LIVE_URL_QUERYSTRING_PARAM', None)
+    if not url_param:
```

Comment:
Not required, it has a default of "live-url" so we know it always exists. You probably had issues with the tests because you weren't using the config settings which get_cms_setting does.